appservice/src/extensionVariables.ts

```diff
@@ -35,6 +40,7 @@ export let ext: IAppServiceExtensionVariables = new UninitializedExtensionVariab
 /**
  * Call this to register common variables used throughout the AppService package.
  */
-export function registerAppServiceExtensionVariables(extVars: IAppServiceExtensionVariables): void {
+export function registerAppServiceExtensionVariables(extVars: IAppServiceExtensionVariables & UIExtensionVariables): void {
     ext = extVars;
+    registerUIExtensionVariables(extVars);
```

Comment:
Is this only to fix the npm link issue or is there another reason for this?

Reply:
Yep I did it to fix the npm link issue and I can't think of any downside